### ISSUE TITLE
Bug – Sort It: Help screen doesn't close on key press, doesn't prevent players from joining while open

### ISSUE THREAD

**Issue by MarioLiebisch** [letsgamedev/Suffragium#69]
If you start the game for the first time (or haven't played any round), the help screen is shown automatically.
It can be closed by clicking on it, but that's not necessary obvious.

Now if you have your hands on WASD anyway and hit any of those buttons, you're automatically adding "Player 1" without closing the help menu. You can see stuff is changing behind, though.

If you happen to click the now visible "Play" button trying to close the help screen, you'll launch the game, too.

**Comment by RedstoneMedia** [comment]
I will create a pull request for this once #68 is merged.
I already fixed the bugs locally, however I'm still not quite sure how I would make the exiting of the pop-up more clear (There is no space in the top right corner for a close button for instance and escape won't work because of the pause menu)

**Comment by Numenter** [comment]
This would prevent the PauseMenu from opening.
```
if event.physical_scancode == KEY_ESCAPE:
    get_tree().set_input_as_handled()
```

**Comment by RedstoneMedia** [comment]
Ah okay nice. I didn't know you could stop event's from propagating.

**Comment by RedstoneMedia** [comment]
I think this issue can be closed, as the issue was fixed in #74

**Comment by MarioLiebisch** [comment]
Ah, missed that! For your next pull request use a phrase such as `fixes #74` and GitHub should automatically notice and close the issue upon merge (mentioning alone won't be enough).
